README.md

```diff
@@ -399,6 +399,10 @@ By default, the database is cached on the local filesystem in the directory `$XD
 
 You can set the cache directory path using the environment variable `GRYPE_DB_CACHE_DIR`.
 
+#### Data staleness
+
+Grype needs up-to-date vulnerability information to provide good matches. By default, it will fail execution if the local database was not updated in the last 5 days. The data staleness check is configurable via the field `staleness-threshold`, under `db`. It uses golang's time duration syntax. Set it to 0 to disable staleness check.
```

Comment:
>It uses golang's time duration syntax

It'd be helpful to link to where this is described, in case the user isn't familiar with this syntax.

Reply:
Agreed, I added the link